android.md

```diff
@@ -35,8 +35,11 @@ Dagger assumes that users on Android will use ProGuard.
 
 ## Recommended ProGuard Settings
 
-Watch this space for ProGuard settings that are relevant to applications using
-Dagger.
+Dagger uses [Error Prone](https://github.com/google/error-prone) annotations. If you don't use it, you may want add the following to your proguard configuration:
```

Comment:
Will using ErrorProne result in this not being unknown? It sounds like to me that this warning will still come up.

Should we be packaging this in the .aar?

Reply:
> Will using ErrorProne result in this not being unknown? It sounds like to me that this warning will still come up.

Using error-prone means this will not trigger a warning, yes.

> Should we be packaging this in the .aar?

Yes! And then this change does not need to be made to the docs and the user doesn't have to know or care in either case.

Reply:
@JakeWharton in that case this whole section should be removed then. I did this change because there was a section about `proguard`. 

The best would be to package `consumerProguardFiles` into `.aar` and remove this section completely.